pdudaemon/listener.py

```
@@ -87,6 +90,26 @@ async def process_request(args, config, daemon):
             return False
         args.hostname = config["aliases"][args.alias]["hostname"]
         args.port = config["aliases"][args.alias]["port"]
+    if args.group:
+        if args.hostname or args.port or args.alias:
+            logger.error("Trying to use hostname or alias or port, when also using group")
+            return False
+        aliases_in_group = []
+        for alias in config.get('aliases', {}).keys():
+            if config["aliases"][alias].get('group', False):
+                aliases_in_group.append(alias)
+        if aliases_in_group:
+            # add job for all the aliases
+            for alias in aliases_in_group:
+                new_args = Args()
+                new_args.request = args.request
+                new_args.delay = args.delay
+                new_args.hostname = config["aliases"][alias]["hostname"]
+                new_args.port = config["aliases"][alias]["port"]
+                await process_request(new_args, config, daemon)
```

This ignores any error. Perhaps it should abort on the first error?